helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -277,11 +277,9 @@ processors:
           tag_name: splunk.com/exclude
           from: pod
         - key: splunk.com/index
-          tag_name: com.splunk.index
-          from: namespace
-        - key: splunk.com/index
-          tag_name: com.splunk.index
           from: pod
+        - key: splunk.com/index
+          from: namespace
```

Comment:
Why we need `k8s.namespace.labels.splunk.com/index` instead of setting `com.splunk.index` directly?

Reply:
in order to guarantee that pod annotation gets the precedence over namespace annotation.

Reply:
It should be already guaranteed. Once attribute from pod is set [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/processor.go#L123), the same attribute from namespace will be _inserted_ [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/processor.go#L133) which is noop if the same key was set from pod

Reply:
We just need to flip the order here. Please submit another PR for that fix to keep git history clean

Reply:
okay great. will revert it.